docs/src/modules/components/LocalizationTable.js

```diff
@@ -0,0 +1,155 @@
+/* eslint-disable material-ui/no-hardcoded-labels */
+import * as React from 'react';
+import PropTypes from 'prop-types';
```

Comment:
Would it work with the component in TypeScript?

```suggestion
```

Reply:
I tried with TypeScript but docs element were complaining that they were not able to find it. Did not investigated further.